pkg/adapter/apiserver/adapter.go

```diff
@@ -146,7 +148,18 @@ func (a *adapter) Start(stopCh <-chan struct{}) error {
 			d.addControllerWatch(gvrc.GVR)
 		}
 
-		reflector := cache.NewReflector(lw, &unstructured.Unstructured{}, d, resyncPeriod)
+		var store cache.Store
+		if gvrc.OwnerApiVersion != "" || gvrc.OwnerKind != "" {
+			store = &controller{
+				apiVersion: gvrc.OwnerApiVersion,
```

Comment:
is it ok to have a "" here when the OR happens to hit on and not the other? Or are they both required?

Reply:
They are both optional. 

- when both are empty, the filtering is disabled
- when only one is specified, the second one behaves like a wildcard